contracts/Router.sol

```diff
@@ -75,24 +70,25 @@ contract Router is ReentrancyGuard {
 	uint256 public oldReserveBalance;
 	bool public hasTransferredOldReserveBalance;
 
-	I_ERC20 public tree = I_ERC20(TREE);
-	I_ERC20 public reserveToken = I_ERC20(DAI);
-	I_TREERewards public lpRewards;
+    I_ERC20 public tree = I_ERC20(TREE);
+    I_ERC20 public reserveToken = I_ERC20(DAI);
+    I_TREERewards public lpRewards;
 
 	uint256 public treeSupply;
 	uint256 public targetPrice;
```

Comment:
We should set a range of acceptable ranges of `targetPrice` and `targetPriceMultiplier`

Reply:
`targetPrice` can never be changed. (It is only set it in the constructor.)  It's there if we need to upgrade or redeploy so we can continue with the price where it was before.

I will add an acceptable range for the targetPriceMultiplier

Reply:
Added in https://github.com/carlfarterson/tree-contracts/pull/4/commits/5fbb24bf413b9ff67cc65c9756a4e5e36b79b59c